gtsfm/frontend/retriever/retriever_base.py

```diff
@@ -0,0 +1,64 @@
+"""Image retriever for front-end.
+Authors: Travis Driver, Jon Womack
+"""
+import abc
+from typing import List, Optional, Tuple, Dict
+import itertools
+
+import numpy as np
+import cbir
+import dask
+from dask.delayed import Delayed
+from gtsam import Pose3, Cal3Bundler
+import networkx as nx
```

Comment:
networkx, Cal3Bundler not needed. running flake8 will catch these locally (CI also will):

```bash
flake8 --max-line-length 120 --ignore E201,E202,E203,E231,W291,W293,E303,W391,E402,W503,E731 gtsfm
```

Reply:
Fixed, but I left networkx in the VocabRetriever, because it is used.